tools/ci_build/build.py

```diff
@@ -358,6 +359,12 @@ def parse_arguments():
     parser.add_argument(
         "--build_micro_benchmarks", action='store_true',
         help="Build ONNXRuntime micro-benchmarks.")
+    parser.add_argument(
+        "--include_ops_by_model", type=str,
+        help="include ops from model(s) under designated path.")
+    parser.add_argument(
+        "--include_ops_by_csv", type=str,
```

Comment:
I would not include the format (csv) of the file in the option name.

Reply:
How about "include_ops_by_file"?